Podfile

```diff
@@ -19,7 +19,7 @@ def uisdkPods
   pod 'PhoneNumberKit', '3.3.1'
   #pod 'KarhooSDK'
   #pod 'KarhooSDK', :path => '../karhoo-ios-sdk'
-  pod 'KarhooSDK', :git => 'git@github.com:karhoo/karhoo-ios-sdk.git', :branch => 'develop'
+  pod 'KarhooSDK', :git => 'git@github.com:karhoo/karhoo-ios-sdk.git', :branch => 'MOB-3066_add_trip_info_meta'
```

Comment:
Once the work on the NSDK is done, it would be good to set this back to develop. Best not to merge when pointing away from develop or latest release versions.

Other than that, LGTM. Just needs this one small change.

Reply:
Done